### DISCUSSION TITLE
This looks awesome !

### DISCUSSION THREAD

**Discussion by astahmer** [kuma-ui/kuma-ui#173] [General]
Hey, I stumbled on this repository after seeing someone that I follow on GH starred it and I'm really impressed by the work you've done ! It seems we try to achieve pretty much the same thing: having the Chakra DX while removing the runtime CSS.

I just wanted to say hi and congratulations. 🐻‍❄️🙏

**Comment by poteboy** [comment]
@astahmer 
Thanks for reaching out and for your kind words! I've looked at Panda CSS as well and I must say I am really impressed by the work you and the Chakra UI team are doing 👍
It's such an amusing coincidence that our projects' namesakes are both bear species! 🐼🐻‍❄️ I was actually super surprised when I found out about this xD
Our libraries do indeed have similar goals, even if our approaches might be a bit different. Panda CSS has definitely served as a valuable source of inspiration for us in creating Kuma UI.
We're actually planning on releasing some big updates for Kuma UI next Monday. We'll be announcing it on [Twitter](https://twitter.com/kuma__ui), so we would be thrilled if you could check it out when it's live! 🙌

Keep up the great work with Panda CSS, and let's keep pushing forward the state of styling libraries together.
